### PR TITLE
TINKERPOP-2081: Fix PersistedOutputRDD to eager persist RDD

### PR DESCRIPTION
**Pull request by artem-aliev**
call rdd.count() action to trigger the caching

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -36,6 +36,7 @@ This release also includes changes from <<release-3-2-10, 3.2.10>>.
 
 ==== Bugs
 
+* TINKERPOP-2081: Fix PersistedOutputRDD to eager persist RDD
```
Comment (spmallette) [inline CHANGELOG.asciidoc]: could you please move this to the 3.3.5 section - 3.3.4 is already released. also, you don't need to add it in the "Bugs" section with JIRA issue...we generate that on release. You can just add a bullet with a description of the change.

Comment (artem-aliev) [reply CHANGELOG.asciidoc]: fixed.

**Comment by spmallette** [comment]
All tests pass with `docker/build.sh -t -n -i`

VOTE +1

**Comment by jorgebay** [comment]
lgtm, VOTE +1

**Comment by dkuppitz** [comment]
VOTE +1

**Pull request merged**